language/move-prover/bytecode-to-boogie/test_mvir/test3.mvir

```diff
@@ -1,3 +1,5 @@
+// FLAGS: -B=-noinfer -B=-noVerify
```

Comment:
I presume these are Boogie flags. Typically you want to always keep `-noinfer` since otherwise Boogie will by default try to do some basic inference of invariants. This functionality is I think rarely used and hence kind of deprecated I believe, and I am not even sure if it still works properly.

Reply:
Good hint! I put this now into the default boogie flags.